app/models/integrations/base/show.rb

```diff
@@ -0,0 +1,50 @@
+module Integrations
+  module Base
+    class Show
```

Comment:
The show gives the handle on the episodes. The production side: baseline unfiltered, sorted set of episodes.